admin-console/src/main/webapp/WEB-INF/apsadmin/jsp/entity/entity-type-delete-confirm.jsp

```diff
@@ -1,33 +1,53 @@
-		<s:text name="%{'title.' + entityManagerName + '.management'}" /></a>
-		&#32;/&#32;
+<ol class="breadcrumb page-tabs-header breadcrumb-position">
+	<li>
+		<s:text name="%{'title.' + entityManagerName + '.management'}"/>
+	</li>
+	<li class="page-title-container">
 		<s:text name="title.entityTypes.editType.remove" />
-	</span>
+	</li>
+</ol>
+<h1 class="page-title-container">
+	<s:text name="title.entityTypes.editType.remove"/>&#32;<s:text name="%{'title.' + entityManagerName + '.management'}"/>
+	<span class="pull-right">
+        <a tabindex="0" role="button" data-toggle="popover" data-trigger="focus" data-html="true" title=""
+		   data-content="TO be inserted" data-placement="left" data-original-title=""><i class="fa fa-question-circle-o"
+																						 aria-hidden="true"></i></a>
+    </span>
 </h1>
+<div class="text-right">
+	<div class="form-group-separator"></div>
+</div>
+<br>
 
-<s:form action="removeEntityType">
-	<p class="sr-only">
-		<wpsf:hidden name="entityManagerName" />
-		<wpsf:hidden name="entityTypeCode" />
-	</p>
-	<div class="alert alert-warning">
+
+<div class="text-center">
+	<s:form action="removeEntityType">
+		<p class="sr-only">
+			<wpsf:hidden name="entityManagerName" />
+			<wpsf:hidden name="entityTypeCode" />
+		</p>
+		<i class="fa fa-exclamation esclamation-big" aria-hidden="true"></i>
+		<p class="esclamation-underline">
+			<s:text name="title.entityTypes.editType.remove"/>&#32;<s:text name="%{'title.' + entityManagerName + '.management'}"/>
+		</p>
 		<p>
 			<s:text name="note.entityTypes.deleteType.areYouSure" />:&#32;
-			<code><s:property value="entityTypeCode" /><code>
-			<s:property value="%{getEntityPrototype(entityTypeCode).typeDescr}" />?
+			<code><s:property value="entityTypeCode" />
+					<s:property value="%{getEntityPrototype(entityTypeCode).typeDescr}" />?
+			</code>
 		</p>
 		<div class="text-center margin-large-top">
-		<wpsf:submit type="button" cssClass="btn btn-warning btn-lg">
-			<span class="icon fa fa-times-circle"></span>&#32;
-            <s:text name="label.remove" />
-        </wpsf:submit>
-		<a class="btn btn-link" href="<s:url action="initViewEntityTypes" namespace="/do/Entity"><s:param name="entityManagerName"><s:property value="entityManagerName" /></s:param></s:url>">
-		<s:text name="note.goToSomewhere" />: <s:text name="%{'title.' + entityManagerName + '.management'}" /></a>
+			<wpsf:submit type="button" cssClass="btn btn-danger button-fixed-width">
+				<s:text name="label.delete"/>
+			</wpsf:submit>
+		</div>
+		<div class="text-center margin-large-top">
+			<a class="btn btn-link" href="<s:url action="initViewEntityTypes" namespace="/do/Entity"><s:param name="entityManagerName"><s:property value="entityManagerName" /></s:param></s:url>">
+				<s:text name="note.goToSomewhere" />: <s:text name="%{'title.' + entityManagerName + '.management'}" />
```

Comment:
Mantenete questa struttura in cui il tasto di delete viene posizionato sulla destra ed il tasto back sulla sinistra.  
 ```
            <div class="text-center margin-large-top">
                <a class="btn btn-default button-fixed-width" href="<s:url action="viewTree" namespace="/do/Page"/>" ><s:text name="label.back" /></a>
                <wpsf:submit type="button" action="delete" cssClass="btn btn-danger button-fixed-width">
                    <s:text name="label.remove" />
                </wpsf:submit>
            </div>
```